frontend/src/Frontend/Wallet.hs

```diff
@@ -282,7 +282,7 @@ getBalances model accounts = do
     accountBalanceReq acc = "(coin.get-balance " <> tshow (unAccountName acc) <> ")"
     mkReqs :: (NetworkName, PublicMeta) -> Accounts key -> Performable m (IntMap (SomeAccount key, Maybe NetworkRequest))
     mkReqs meta someaccs = for someaccs $ \sa ->
-      fmap (sa,) . traverse (mkReq meta) $ activeAccountOnNetwork (fst meta) sa
```

Comment:
This has a bug where it doesn't show balances for the non default wallet. This change reverts the smarts attempted in #272